### PR TITLE
Fixed issued where md file would not render image since directory was…

### PR DESCRIPTION
**Pull request by densellp**
There was an issue where the image file would not render since the directory for the images was not being opened properly. A simple change was made to remove to `..` at the beginning to render the image.

### CODE REVIEW

File: docs/4-software-development-practices/4.3.3-github.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ After a pull request is created in GitHub it can be reviewed by viewing a side-b
 
 ### Peer Reviews: A Sacred Space for Developers
 
-![Peer Previews](../img/git-pr.webp ':class=img-center')
+![Peer Previews](/img/git-pr.webp ':class=img-center')
```
Comment (baserrato) [inline docs/4-software-development-practices/4.3.3-github.md:17]: Will work for markdown but docsify still breaks based on the relative path it references the image from

File: docs/4-software-development-practices/4.3.3-github.md [hunk 2]
```diff
@@ -14,13 +14,13 @@ After a pull request is created in GitHub it can be reviewed by viewing a side-b
...
 
 Apart from quality gates, PRs provide an opportunity to perform honest, robust, and friendly code reviews. It is a sacred space often utilized only by the developers, as it narrows the audience to low-level code reviews.
 
 During PRs, developers will work together to ensure that the code meets the business value promised by the work item. Common discussions will revolve around scope creep, optimizing implementation, and alignment on the team's code norms.
 
-![Pull Requests](../img/git-peer-review.webp ':class=img-shadow-center')
+![Pull Requests](/img/git-peer-review.webp ':class=img-shadow-center')
```
Comment (baserrato) [inline docs/4-software-development-practices/4.3.3-github.md:23]: Same issue as line 17

**Review by baserrato — CHANGES_REQUESTED** [review]

File: docs/4-software-development-practices/4.3.3-github.md (hunk 1 above)
Comment (densellp) [reply docs/4-software-development-practices/4.3.3-github.md]: changed completed

File: docs/4-software-development-practices/4.3.3-github.md (hunk 2 above)
Comment (densellp) [reply docs/4-software-development-practices/4.3.3-github.md]: changed completed

**Comment by densellp** [comment]
Changes made to img file path

**Pull request merged**